TwitchDownloaderCore/TwitchObjects/ChatBadge.cs

```diff
@@ -19,26 +19,43 @@ public enum ChatBadgeType
         PrimeGaming = 128
     }
 
+    public class ChatBadgeSKBitmapData
+    {
+        public string title { get; set; }
+        public object description { get; set; }
+        public SKBitmap bitmap { get; set; }
+    }
```

Comment:
I don't exactly see why there are now separate `byte[]` and `SkBitmap` classes. It's not like its saving memory by creating only one or the other.

Reply:
I re-done this part, it now only has `ChatBadgeData` class, and changed `versions` back to `Dictionary<string, SkBitmap>`.